gsoc-2024-proxy/pom.xml

```diff
@@ -30,6 +30,15 @@
       <groupId>io.vertx</groupId>
       <artifactId>vertx-web</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.vertx</groupId>
+      <artifactId>vertx-web-proxy</artifactId>
+      <version>4.1.8</version>
```

Comment:
Why using a version which is not the version managed by `vertx-dependencies` ?

Reply:
Apologise for that mistake. Instead of manually injecting the dependency I use what IntelliJ suggested. I corrected it now.